.doppler-ci

```diff
@@ -0,0 +1 @@
+Jenkinsfile
```

Comment:
You need to mark this file as a _symbolic link_ to `Jenkinsfile` file.

![image](https://user-images.githubusercontent.com/1157864/120321338-933e3d00-c2b9-11eb-8a29-ad6fb82ae084.png)

I am not sure how to do that in Windows, you can ask @hrocca or @elsupergomez, they already did it.

Reply:
In Windows open _Bash_ as _Admin_ and execute this on the project path

`export MSYS=winsymlinks:nativestrict; ln -s Jenkinsfile .doppler-ci`